### DISCUSSION TITLE
How to create a drawer component?

### DISCUSSION THREAD

**Discussion by cusxio** [ariakit/ariakit#2881] [Questions] [answered]
Example of a drawer component,
https://chakra-ui.com/docs/components/drawer

Since there isn't a drawer component in `ariakit`, I was wondering if using [`Disclosures`](https://ariakit.org/components/disclosure) is the right way to go?

**Comment by diegohaz** [comment, marked as answer]
That's a [Dialog](https://ariakit.org/examples/dialog-animated) semantically. You can style it as a drawer.

**Reply by cusxio** [reply]
Gotcha!
